DotCMIS/DotCMIS.csproj

```diff
@@ -40,6 +40,10 @@
     <AssemblyOriginatorKeyFile>dotcmis.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="log4net, Version=1.2.13.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\log4net.2.0.3\lib\net40-full\log4net.dll</HintPath>

```

Comment:
now i can see: the problem is that i'm using this project as a submodule of CmisSync, so the resulting path is related to the root project that indeed has the packages folder.
I don't want to add a dll to the project; it need to be managed by NuGet, but i don't know how to solve this